Backend/app/dashboard/Lib/src/Tools/Auth/Api.hs

```diff
@@ -80,16 +79,16 @@ verifyApi ::
   m ApiTokenInfo
 verifyApi requiredAccessLevel token = do
   (personId, merchantId, city) <- Common.verifyPerson token
-  (verifiedPersonId, dashboardAccessType) <- verifyAccessLevel requiredAccessLevel personId
+  verifiedPerson <- verifyAccessLevel requiredAccessLevel personId
   verifiedMerchant <- verifyServer requiredAccessLevel.serverName merchantId
   _ <- verifyCity verifiedMerchant city
   pure
     ApiTokenInfo
-      { personId = verifiedPersonId,
+      { personId = verifiedPerson.id,
```

Comment:
here `personId` is redundant if we have `person`, but too much changes required to remove it, let it be for now